compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -338,7 +340,7 @@ StringRef normalizeAMDGPUTarget(StringRef target) {
     return target;
 
   return llvm::StringSwitch<StringRef>(target.lower())
-      .Cases("mi300x", "mi300a", "gfx942")
+      .Cases("mi300x", "mi300a", "mi308x", "gfx942")
```

Comment:
keep sorted?
```suggestion
      .Cases("mi308x", "mi300x", "mi300a", "gfx942")
```

@kuhar do you want new SKUs to go at the start of the list or the end of the list?

Reply:
I'd think we should append them to keep consistent with `.Cases("cdna3", "gfx940", "gfx941", "gfx942",` above where newer targets are placed towards the end.

Ultimately I don't care as long as we some consistent ordering.